webextension/background.js

```diff
@@ -6,14 +6,136 @@
 
 /* globals browser */
 
-const gURLThankYou = "https://mozilla.github.io/webcompat-blipz-experiment/thanks.html";
+let gCurrentlyPromptingTabId;
+
+const Config = (function() {
```

Comment:
nit: I dunno if having a reference to a class instance starting with Uppercase will pass Firefox JS style... or if that's even defined. I guess we'll let our peer decide. :)

Reply:
I can just preemptively rename it (maybe gConfig?), if you'd like.

Reply:
Ah, let's just go ahead as-is for now; we'll get clearer style feedback from a review peer later.